DOCS-4157 Service Catalog Terraform Update (#15457)
DataDog/documentation: content/en/tracing/service_catalog/setup.md

## Overview

> The Service Catalog displays a list of matching services. The **Registered** column in the **Ownership** tab indicates that a corresponding service is registered with a green check mark.

Reviewer: ```suggestion
The Service Catalog displays a list of services. The **Registered** column in the **Ownership** tab indicates that a corresponding service is registered with a green check mark.
```

After you update the YAML files for your repositories, your changes propagate to the Service Catalog. To prevent accidental overwriting, create and modify your service definition files with the GitHub Apps integration or the [Service Definition API endpoints][1].

## Automate service definition updates with Terraform

> The Service Catalog provides a service definition as a Terraform resource. In order to create and manage services in the Service Catalog through automated pipelines, you need the [Datadog Provider][8] v3.16.0.

Reviewer: ```suggestion
The Service Catalog provides a service definition as a Terraform resource. Creating and managing services in the Service Catalog through automated pipelines requires [Datadog Provider][8] v3.16.0 or later.
```
